src/releaseAndTag.js

```diff
@@ -107,7 +106,7 @@ const createTempDirectory = async (name) => {
   if (exists) {
     throw new Error(`Tried to create directory, but it already exists: ${dir}`);
   }
-  mkdirp(dir);
+  fs.mkdirSync(dir, { recursive: true });
```

Comment:
I don't think this code is exercised in tests so a manual check that this works would be great.

Reply:
Perhaps should be this instead?

```suggestion
  await fs.mkdir(dir, { recursive: true });
```